lib/api/BackbeatRequest.js

```diff
@@ -79,6 +85,14 @@ class BackbeatRequest {
         return this._routeDetails;
     }
 
+    /**
+     * Get the request http verb method
+     * @return {string} request http verb
+     */
+    getHTTPVerb() {
```

Comment:
It would be better to stick to the RFC terminology and call this `getHTTPMethod` instead.